presidio-analyzer/Pipfile

```diff
@@ -4,16 +4,20 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-spacy = ">=3.4.4,<4.0.0"
+spacy = ">=3.4.4, <4.0.0"
 regex = "*"
 tldextract = "*"
 flask = ">=1.1"
 pyyaml = "*"
 phonenumbers = ">=8.12,<9.0.0"
 typing-extensions = "*"
+spacy-huggingface-pipelines = "*"
```

Comment:
Do we need to add this package to the root-level NOTICE file?

Reply:
Same with stanza

Reply:
Good catch, thanks.

Reply:
Done